evergreen.yml

```diff
@@ -104,17 +132,92 @@ functions:
       type: test
       params:
         shell: bash
-        working_dir: mongo-odbc-driver
+        working_dir: mongosql-odbc-driver
         script: |
           ${prepare_shell}
           cargo build --release
 
+  "upload artifact":
+    - command: s3.put
+      params:
+        build_variants:
+          - windows-32
+          - windows-64
+        aws_key: ${aws_key}
+        aws_secret: ${aws_secret}
+        local_file: mongosql-odbc-driver/target/release/mongoodbc.dll
+        remote_file: mongosql-odbc-driver/artifacts/test/${version_id}/${build_variant}/mongoodbc.dll
+        bucket: mciuploads
+        permissions: public-read
+        content_type: application/octet-stream
+
+  "download artifact":
+    - command: s3.get
+      params:
+        build_variants:
+          - windows-32
+          - windows-64
+        aws_key: ${aws_key}
+        aws_secret: ${aws_secret}
+        remote_file: mongosql-odbc-driver/artifacts/test/${version_id}/${build_variant}/mongoodbc.dll
+        local_file: mongosql-odbc-driver/mongoodbc.dll
+        bucket: mciuploads
+
+  "setup driver on Windows":
```

Comment:
Unfortunately, because we don't have the setup routines (the UI for the window that is supposed to pop-up when doing "configure" on ODBC Admin), the command fails with `Add-OdbcDsn : The setup routines for the %s ODBC driver could not be accessed.  Please reinstall the driver.`.